### DISCUSSION TITLE
Active stabilization test

### DISCUSSION THREAD

**Discussion by SupSonAlv** [GuidodiPasquo/Actively-Stabilized-Model-Rocket-Simulator#10] [General]
Only Pitch stabilization can be tested in this application, right?
As far as I can see, there is no add-on for yaw and roll stabilization.

**Comment by GuidodiPasquo** [comment]
Hello!, yes, only pitch stabilization

**Comment by SupSonAlv** [comment]
Hi,
Have you ever simulated a model you created in this application for pitch stabilization and tested a rocket with the same features for real?

**Reply by GuidodiPasquo** [reply]
I haven't, but I know two people that did with TVC and it worked for them.

**Comment by SupSonAlv** [comment]
Maybe I can be the person to test this app with Fins and understand the difference between real and simulator, but I'm a bit skeptical.

Wouldn't that be a problem if you just simulate Pitch stabilization and do real testing?
Because when a rocket climbs upward, both yaw and pitch angles must be kept constant in order for a rocket to climb vertically, and in order to do this better, roll stabilization must be done well. Otherwise, the control may be reversed.

**Reply by GuidodiPasquo** [reply]
If you could that would be great!

Pitch and yaw are identical due to symmetry, so if you find the gains to control pitch, they would work on yaw. Roll stabilization is not too hard to simulate with traditional methods, so you can find the gains using transfer functions.

**Reply by SupSonAlv** [reply]
That's exactly my goal, I'm going to make a nice simulation first and then test it in reality.

I can give you the result.

**Reply by GuidodiPasquo** [reply]
Yes, please!, keep me updated!
